fsw/src/sc_cmds.c

```diff
@@ -552,38 +557,19 @@ void SC_OneHzWakeupCmd(const SC_OneHzWakeupCmd_t *Cmd)
             SC_ServiceSwitchPend();
         }
 
-        if (SC_AppData.NextProcNumber == SC_Process_ATP)
-        {
-            SC_ProcessAtpCmd();
-        }
-        else
-        {
-            if (SC_AppData.NextProcNumber == SC_Process_RTP)
-            {
-                SC_ProcessRtpCommand();
-            }
-        }
+        SC_ProcessAtpCmd();
 
-        SC_UpdateNextTime();
-        if ((SC_AppData.NextProcNumber == SC_Process_NONE) ||
-            (SC_AppData.NextCmdTime[SC_AppData.NextProcNumber] > SC_AppData.CurrentTime))
-        {
-            SC_OperData.NumCmdsSec         = 0;
-            IsThereAnotherCommandToExecute = false;
-        }
-        else /* Command needs to run immediately */
-        {
-            if (SC_OperData.NumCmdsSec >= SC_MAX_CMDS_PER_SEC)
-            {
-                SC_OperData.NumCmdsSec         = 0;
-                IsThereAnotherCommandToExecute = false;
-            }
-            else
-            {
-                IsThereAnotherCommandToExecute = true;
-            }
+        SC_ProcessRtpCommand();
```

Comment:
Discussed updates to this live since NextProcNumber would prevent the current implementation from working as desired.  Adding comment so I can resolve it once updated.